packages/react-component-library/src/components/Timeline/context/types.ts

```diff
@@ -58,7 +58,7 @@ export type TimelineState = {
 }
 
 export const TIMELINE_ACTIONS = {
-  CHANGE_START_DATE: 'CHANGE_START_DATE',
```

Comment:
FYI the actions are public, so we'll probably want to keep `CHANGE_START_DATE` as an alias of `CHANGE_DATE`.